examples/generative/corrdiff/conf/config_generate.yaml

```diff
@@ -89,6 +90,7 @@ dataset:
   gridtype: "sinusoidal"
```

Comment:
It's unclear what gridtypes are supported.

Reply:
Added supported gridtypes